docs/serverless-v2/08-01-expose-lambda.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Expose lambdas with API Rules
+type: Tutorials
+---
+
+This tutorial shows how you can expose a lambda function to access it outside the cluster, through an HTTP proxy. To expose it, you use an APIRule custom resource (CR) managed by the in-house API Gateway Controller. This controller reacts to an instance of the APIRule CR and, based on its details, it creates an Istio Virtual Service and Oathkeeper Access Rules that specify your permissions for the exposed function.
+
+This example is based on a simple code which returns the "Hello, World!" string when you access the HTTP host on which the lambda is exposed. This lambda is not secured by an access token (`handler: noop`), and it accepts `GET`, `POST`, `PUT`, and `DELETE` request types.
+
+>**NOTE:** To learn more about securing your lambda, see [this](/components/api-gateway-v2/#tutorials-expose-and-secure-a-service-deploy-expose-and-secure-the-sample-resources) tutorial.
+
+## Steps
+
+Follows these steps:
+
+1. Export these variables:
+
+    ```bash
+    export DOMAIN={DOMAIN_NAME}
+    export NAME={RESOURCE_NAME}
+    export NAMESPACE="kyma-system"
```

Comment:
`export NAMESPACE="serverless"` `kyma-system` is one of system namespaces, and users shouldn't create resources on system namespaces.